docs/CloudFerry_Beginner_Demo_How-To.md

```diff
@@ -0,0 +1,225 @@
+Introduction
+============
+
+Assumed Knowledge: You should be a Linux/Unix system administrator or
+engineer with at least a passing familiarity with programming languages
+(CF is written in Python), networking, databases, Unix text editors,
+large scale storage appliances, RESTful applications and, of course,
+substantial knowledge of and experience with OpenStack including at the
+command line.
+
+CloudFerry is a highly configurable tool and as such there are more ways
+to use it than can be described in a brief document. This demo will
+assist you in creating a pair of virtual OpenStack clouds, adding
+tenants and configuring resources for you to migrate. 
+
+In the real world it’s almost certain that since migration implys moving
+workloads from one cloud environment to another that you’ll need to
+perform a Condensation and Evacuation run prior to migration simply to
+free up enough hardware to pull it off. Those 2 topics are not covered
+in this document. 
+
+How-To: CloudFerry for Beginners
+================================
+**System Requirements**
+-----------------------
+
+1.  100GB free disk space as a minimum.
+2.  CPU power to spare 5 cores.
+3.  12GB free RAM (you should have 16GB total as a practical minimum)
+4.  Internet access (for git clone).
+5.  MacOS or Ubuntu Linux (or other Debian distribution) Operating
+    System
+
+Setting Up CloudFerry Test Environment on MacOS
+===============================================
```

Comment:
"How-To..." and "Setting Up..." headers have the same level, while it seems "Setting Up..." should be part of "How To..."